### ISSUE TITLE
re-enable FCM

### ISSUE THREAD

**Issue by diracdeltas** [brave/brave-browser#12901]
lot of webcompat issues in this thread: https://github.com/brave/brave-browser/issues/2143#issuecomment-733509235

we should figure out if we can enable it without directly going through google servers

**Comment by EmielJB** [comment]
Like you can choose to ose the shields, you should also be able to choose using FCM as a user. 
I would argue that most of Brave's users are people who have an idea of what they are doing regarding their browsing behaviour, so they should be more than capable of making this choice.

**Comment by pbennett** [comment]
> For example, the live chat software that I use uses FCM to alert when a new chat is available. This is critical for my business.

Agreed.  It's baffling that Brave just decided on their own with users not asking for this to hard-disable push notifications for something that users *already have to accept and allow !!* and that Brave still prompts them for !  It basically means I'd have to tell users - just don't use Brave.

**Comment by arunbasillal** [comment]
> and that Brave still prompts them for

Yeah, good point. That makes no sense really.

**Comment by ThreeDeeJay** [comment]
Could these work as a replacement? 
https://unifiedpush.org/
https://github.com/binwiederhier/ntfy
